proto/src/determined/api/v1/experiment.proto

```diff
@@ -575,3 +575,34 @@ message MoveExperimentRequest {
 
 // Response to MoveExperimentRequest.
 message MoveExperimentResponse {}
+
+// Request to get the list of Searcher Events.
+message GetSearcherEventsRequest {
+  // The id of the experiment.
+  int32 experiment_id = 1;
+}
+
+// Response to GetSearcherEventsRequest.
+message GetSearcherEventsResponse {
+  // The requested list of searcher events.
+  // str value can be: InitialOperations, TrialCreated, ValidationCreated,
+  // TrialExitedEarly, TrialClosed appended with a counter id to represent
+  // order.
+  repeated string searcher_event = 1;
```

Comment:
Doesn't this need to be a message type with other fields in addition to the event type?